npy_array/npy_array/npy_array.cc

```diff
@@ -18,14 +18,14 @@
 #include <cstdint>
 #include <regex>  // NOLINT: ok to use std::regex in third_party code.
 #include <string>
+#include <string_view>
 #include <vector>
 
-#include "glog/logging.h"
 #include "absl/strings/match.h"
 #include "absl/strings/numbers.h"
 #include "absl/strings/str_join.h"
 #include "absl/strings/str_split.h"
-#include "absl/strings/string_view.h"
+#include "glog/logging.h"
```

Comment:
Unless it really bothers you, let's keep its current position... The export from third_party will reset the header order.

Reply:
Moved back.

The rest is all my clang-format, which I think is the same as Google's internal one.